content/this-month/2022-08/index.md

```diff
@@ -55,6 +55,17 @@ If you maintain a Rust project related to operating system development and are l
 
 In this section, we describe updates to Rust OS projects that are not directly related to the `rust-osdev` organization. Feel free to [create a pull request](https://github.com/rust-osdev/homepage/pulls) with the updates of your OS project for the next post.
 
+### ['cdrzewiecki/celos'](https://gitlab.com/cdrzewiecki/celos)
```

Comment:
Minor formatting suggestion (backticks instead of single quotes):

```suggestion
### [`cdrzewiecki/celos`](https://gitlab.com/cdrzewiecki/celos)
```

Reply:
Good catch, thanks!